src/intervals/arithmetic.jl

```diff
@@ -218,10 +218,10 @@ function extended_div(a::Interval{T}, b::Interval{T}) where T<:Real
     S = typeof(a.lo / b.lo)
     if 0 < b.hi && 0 > b.lo && 0 ∉ a
         if a.hi < 0
-            return (Interval(T(-Inf), a.hi / b.hi), Interval(a.hi / b.lo, T(Inf)))
+            return (@round(T(-Inf), a.hi / b.hi), @round(a.hi / b.lo, T(Inf)))
```

Comment:
This will work, but since no rounding is required for `T(Inf)`, it will be more efficient to directly use the rounded form of `/`.